CHANGELOG.md

```diff
@@ -43,7 +44,7 @@ initialized=true when a Raft join is in progress. [[GH-10498](https://github.com
 SECURITY:
 
 * LDAP Auth Method: We addressed an issue where error messages returned by the
-  LDAP auth methold allowed user enumeration [[GH-10537](https://github.com/hashicorp/vault/pull/10537)]. This vulnerability affects Vault OSS and Vault 
+  LDAP auth method allowed user enumeration [[GH-10537](https://github.com/hashicorp/vault/pull/10537)]. This vulnerability affects Vault OSS and Vault 
```

Comment:
Same question re: https://github.com/hashicorp/vault/blob/release/1.6.x/changelog/10537.txt

Though for this one 1.6.1 is already released so I'm unsure of the correct approach here.

Reply:
I didn't create that one, but, sure, I can add it. I didn't even know this changelog folder was a thing...